source/api_documentation/transactions_api/_advertiser_param_table.rst

```diff
@@ -0,0 +1,239 @@
+
+  * - corrects_transaction_id
+    - Corrects Call
+    - [Correction only] Id of the original transaction that this transaction updates. Values in this row are the corrected ones and should replace the original values. Same format as transaction_id. Up-to 32 character string, can contain alphanumeric characters (i.e. 0-9A-Z) and the "-".
+
+  * - destination_phone_number
+    - Destination Phone Number
+    - The phone number where the call was transferred to (useful if an IVR transfers to multiple destinations). Up-to 20 character string, can contain numeric characters (i.e. 0-9) and the following additional characters: '-', '#', '*', 'x', and ','. '
+
+  * - duration
+    - Total Duration (HH:MM:SS)
+    - Duration of the call in seconds. Includes any time spent in an IVR tree before transferring to the call center.
+
+  * - ivr_duration
+    - IVR Duration (HH:MM:SS)
+    - Duration in seconds that the call spent in the IVR tree.
+
+  * - keypress_1
+    - Key 1
+    - Name of the first key that was pressed
+
+  * - keypress_2
+    - Key 2
+    - Name of the second key that was pressed
+
+  * - keypress_3
+    - Key 3
+    - Name of the third key that was pressed
+
+  * - keypress_4
+    - Key 4
+    - Name of the fourth key that was pressed
+
+  * - keypresses
+    - Keypresses
+    - List of unique keynames that were pressed during the call
+
+  * - matching_advertiser_payin_policies
+    - Matching Advertiser Payin Policies
+    - List of advertiser policies that matched (base, bonus1, bonus2, etc.) to determine the advertiser payin, separated by +. For example, base+bonus1+bonus3. Note that if there was any advertiser payin, this field guaranteed to start with base.
+
+  * - media_type
+    - Media Type
+    - Media type of the transaction source
+
+  * - mobile
+    - Phone Type
+    - "Landline" or "Mobile" or empty string if type is unknown
+
+  * - notes
+    - Notes
+    - Free-form notations on transaction
+
+  * - opt_in_SMS
+    - Opt In Sms
+    - Whether the caller opted in to receive an SMS promotion.
+
+  * - original_order_id
+    - Order ID
+    - [Sales reporting only] Id of the original transaction that this row is in reference to. Up-to 32 character string, can contain alphanumeric characters (i.e. 0-9A-Z) and the "-".
+
+  * - payin_conditions
+    - Payin Conditions
+    - Base condition with { highlighting } around the term(s) that disqualified advertiser payin. For example: duration > 1 min and {in_region}
+
+  * - qualified_regions
+    - Qualified Regions
+    - The list of regions that that the caller matched
+
+  * - recording
+    - Recording
+    - URL to the call recording, if available
+
+  * - region
+    - Region
+    - Region (state, province or country) where transaction originated
+
+  * - repeat_calling_phone_number
+    - Repeat Caller
+    - Whether the call was a repeat call. Repeat call detection is not applied to shared or unavailable caller ids.
+
+  * - signal_10_value
```

Comment:
here too.. let's put signal_10 at the end, even though it won't be strict alphabetical